src/main/java/basic/Exercise15.java

```diff
@@ -0,0 +1,14 @@
+package basic;
+
+public class Exercise15 {
+
+    public int findNumBit(int num) {
+        StringBuilder result = new StringBuilder("");
+
+        while (num > 0) {
```

Comment:
Bài này thuần tính toán trên bit, không dùng string nha

Reply:
Done